sympy/matrices/tests/test_standalone_funcs.py

```diff
@@ -0,0 +1,91 @@
+from sympy import PurePoly, Symbol

```

Comment:
This file should probably be called `test_determinant.py` if it is testing the functions from the `determinant` module.

Reply:
Maybe it makes sense to move the relevant tests out of `test_matrices`

Reply:
`test_matrices.py` is too big IMHO, it should be split up a little?

Reply:
Yes it can be split up. This module should just test the functions in the determinant module. The relevant tests from test_matrices should come here. Then test_matrices should just test that the methods still work.